databricks/koalas/series.py

```diff
@@ -3409,6 +3409,80 @@ def replace(self, to_replace=None, value=None, regex=False) -> 'Series':
 
         return self._with_new_scol(current)
 
+    def where(self, cond, other=np.nan):
```

Comment:
@itholic seems like pandas shares the same implementation internally. After this PR is merged, can you move this into `_Frame` class and implement `DataFrame.where` as well?

Reply:
okay, i'm going to work right after this PR is merged